app/controllers/events/import.js

```diff
@@ -0,0 +1,65 @@
+import Controller from '@ember/controller';
+import { run } from '@ember/runloop';
+import ENV from 'open-event-frontend/config/environment';
+
+export default Controller.extend({
+  importStatus   : '',
+  importError    : '',
+  btnImportEvent : false,
+  file           : false,
+  fileName       : '',
+  importTask(taskUrl) {
+    let _this = this;
```

Comment:
This is not required. You can just use `this` within [Arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions). The context is binded correctly.